src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,28 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab, idx) => (
+          <li
+            className={classNames({ 'is-active': tab.id === selectedTab.id })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#tab-${idx + 1}0`}
+              data-cy="TabLink"
+              onClick={() => tab.id !== selectedTab.id && onTabSelected(tab)}
+            >
+              {`Tab ${idx + 1}0`}
```

Comment:
please, update to 1,2,3
![image](https://github.com/mate-academy/react_tabs-js/assets/91946036/d0c265de-01ef-49fc-b49f-91bbd91d2cc2)

instead of {`Tab ${idx + 1}0`} use  {title}